examples/plot_connectivity.py

```diff
@@ -38,9 +38,19 @@
 # Instantiating the network comes with a predefined set of connections that
 # reflect the canonical neocortical microcircuit. ``net.connectivity``
 # is a list of dictionaries which detail every cell-cell, and drive-cell
-# connection.
+# connection. The weights of these connections can be visualized with
+# :func:`~hnn_core.viz.plot_connectivity_weights` as well as
+# :func:`~hnn_core.viz.plot_cell_connectivity`
+from hnn_core.viz import plot_connectivity_weights, plot_cell_connectivity
 print(len(net_erp.connectivity))
-print(net_erp.connectivity[0:2])
+
+conn_idx = 15
+print(net_erp.connectivity[conn_idx])
+plot_connectivity_weights(net_erp, conn_idx)
+
+conn_idx, gid_idx = 20, 11
+src_gid = net_erp.connectivity[conn_idx]['src_range'][gid_idx]
+fig, ax = plot_cell_connectivity(net_erp, conn_idx, src_gid)
```

Comment:
any reason you used two different `conn_idx` ? Might be good to see the same connectivity with two different plots. What is the units in X/Y axis? Also missing colorbar?

Reply:
Unfortunately units are arbitrary until we settle on physical units. But internally they are just the coordinates stored in `net.pos_dict`.

Also I'll update according to these suggestions.